bookshelter-app/src/components/BookCardList/index.tsx

```diff
@@ -1,27 +1,19 @@
 import BookCard from '../BookCard';
 import styles from './index.module.css';
 import { Book } from '../../types';
-import { useNavigate } from 'react-router-dom';
+import { Link } from 'react-router-dom';
 
 interface BooksProps {
   books: Book[];
 }
 
 const BookCardList = ({ books }: BooksProps) => {
-  const navigate = useNavigate();
-
-  const onHandleClick = (book: Book) => {
-    navigate(`/detail/${book.id}`);
-  };
-
   return (
     <div className={styles.bookList}>
       {books.map((book) => (
-        <BookCard
-          key={book.id}
-          book={book}
-          onHandleClick={() => onHandleClick(book)}
-        />
+        <Link key={book.id} to={`/detail/${book.id}`}>
+          <BookCard key={book.id} book={book} />
```

Comment:
remove key in Book

Reply:
if remove key console Warning: Each child in a list should have a unique "key" prop.

Reply:
Link already has the key.